packages/mapviewer/src/modules/i18n/locales/de.json

```
@@ -110,6 +110,7 @@
     "description": "Beschreibung",
     "description_instructions": "Beschreibung des WMS Layers",
     "desktop_redirect": "Link zur Desktop Version",
+    "display_on_map": "Anzeige auf der Karte",
```

I think better would be:
```suggestion
    "display_on_map": "Auf der Karte anzeigen",
```

---

German language: "let's tease people and put verbs at the end. sometimes."